File: C-Sharp-Part-1/3. Operators and Expressions/Problem07/PointInACircle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problem7
{
    class PointInACircle
    {
        static void Main(string[] args)
        {
            double x = Double.Parse(Console.ReadLine());
            double y = Double.Parse(Console.ReadLine());
            if (Math.Pow((x - 0), 2) + Math.Pow((y - 0), 2) <= Math.Pow(2, 2))
            {
                Console.WriteLine("yes {0:F2}", Math.Sqrt(Math.Pow((x-0),2) + (Math.Pow((y-0),2))));
            }
            else
            {
                Console.WriteLine("no {0:F2}", Math.Sqrt(Math.Pow((x-0),2) + (Math.Pow((y-0),2))));
            }
        }
    }
}

File: JavaScript-Fundamentals/04. Operators and Expressions/Problem06/PointInACircle.js
function pointInCircle(arr) {
    var x = arr[0];
    var y = arr[1];
    var condition = Math.pow((x - 0), 2) + Math.pow((y - 0), 2);
    if (condition <= Math.pow(2, 2)) {
        console.log('yes ' + Math.sqrt(condition).toFixed(2));
    } else {
        console.log('no ' + Math.sqrt(condition).toFixed(2));
    }
}